### PR TITLE
Import jk

### PR DESCRIPTION
**Pull request by juhis**
Matrix import faster

### CODE REVIEW

File: pheweb/load/external_matrix.py [hunk 1]
```diff
@@ -67,6 +72,12 @@ def run(argv):
     parser.add_argument('--ref', default="ref", action='store', type=str, help='ref column name in result files')
     parser.add_argument('--alt', default="alt", action='store', type=str, help='alt column name in result files')
     parser.add_argument('--other_fields', action='store', type=str, help='comma separated list of other column names in result files')
+    parser.add_argument('--require_match', dest="require_match", action='store_true', help='if given, require a variant to match between the sites and pheno files for it to be written')
+    parser.add_argument('--no_require_match', dest="require_match", action='store_false', help='if given, dont require a variant to match between the sites and pheno files for it to be written')
```
Comment (Fedja) [inline pheweb/load/external_matrix.py]: What is this? Why have two switches for binary choice? Make a default and have a switch the alter that choice!

File: pheweb/load/external_matrix.py [hunk 2]
```diff
@@ -149,11 +160,12 @@ def run(argv):
...
                     out.write("\t".join(linedat) + "\n")
                 linedat.clear()
 
-    subprocess.check_call(["bgzip", args.path_to_res + "matrix.tsv" ])
-    subprocess.check_call(["tabix","-s 1","-e 2","-b 2", args.path_to_res + "matrix.tsv.gz" ])
+    if args.tabix:
+        subprocess.check_call(["bgzip", args.path_to_res + "matrix.tsv" ])
+        subprocess.check_call(["tabix","-s 1","-e 2","-b 2", args.path_to_res + "matrix.tsv.gz" ])
 
 run("asd")
```
Comment (Fedja) [inline pheweb/load/external_matrix.py]: This is my own "asd" but let's remove it now that I spot it

File: wdl/import.wdl [hunk 3]
```diff
@@ -89,31 +89,60 @@ task matrix {
...
+        for p in processes:
+            if p.poll() is None:
+                p.wait()
+        KARMES
+
+        cmd="paste <(cat ${sites} | sed 's/chrom/#chrom/') "
+        for file in *pheno_piece.matrix.tsv; do
+            cmd="${dollar}cmd <(cut -f5- ${dollar}file) "
+        done
+        echo ${dollar}cmd | bash > generated-by-pheweb/matrix.tsv
```
Comment (Fedja) [inline wdl/import.wdl]: Is there a need for a separate bash shell? WHy not just run the command ${dollar}cmd > generated-by-pheweb/matrix.tsv

Comment (juhis) [reply wdl/import.wdl]: No need indeed

File: wdl/import.wdl [hunk 4]
```diff
@@ -123,7 +152,7 @@ task matrix {
         File top_hits_tsv = "pheweb/generated-by-pheweb/top_hits.tsv"
         File top_hits_1k = "pheweb/generated-by-pheweb/top_hits_1k.json"
         File pheno_gene = "pheweb/generated-by-pheweb/best-phenos-by-gene.json"
-        Array[File] tmp = glob("pheweb/generated-by-pheweb/tmp/*")
+        #Array[File] tmp = glob("pheweb/generated-by-pheweb/tmp/*")
```
Comment (Fedja) [inline wdl/import.wdl]: Can the comment be removed?

Comment (juhis) [reply wdl/import.wdl]: I removed the # in a subsequent commit

**Pull request merged**